Blackjack Game/README.md

```diff
@@ -0,0 +1,32 @@
+# Blackjack game
+
```

Comment:
Add a small description of the project please!

Reply:
done